packages/flutter/test/painting/system_fonts_test.dart

```diff
@@ -283,8 +283,8 @@ void main() {
     final RenderObject renderObject = tester.renderObject(
       find.descendant(
         of: find.byKey(const Key('parent')),
-        matching: find.byType(CustomPaint),
-      ).first,
+        matching: find.byKey(const ValueKey<String>('time-picker-dial')),
```

Comment:
What was this change for? Why are tests in `painting` dependent on the Time Picker?

Reply:
This was just to make the test less fragile: when I changed the structure of the time picker, it broke this test, but it could have been using the key (that the time picker already had) to find the widget instead of the structure of the widget tree.

I'm not sure why it's in painting, but presumably it is a test meant to check for a regression in painting fonts.